src/parsers/junit.js

```diff
@@ -76,10 +76,12 @@ function getTestResult(json) {
   result.total = result.total - result.skipped;
   result.passed = result.total - result.failed - result.errors;
   result.duration = rawResult["@_time"] * 1000;
-  const rawSuites = rawResult["testsuite"];
-  const filteredSuites = rawSuites.filter(suite => suite.testcase);
-  for (let i = 0; i < filteredSuites.length; i++) {
-    result.suites.push(getTestSuite(filteredSuites[i]));
+  if (result.total > 0 && result.skpped == 0) { // Don't filter if there are no tests
```

Comment:
- `result.skpped` is a typo
- there are chances that a test suite can contain skipped tests along with the others.

Reply:
Fixed compare and verified unit tests pass with changes.